### DISCUSSION TITLE
An Icon or a CheckBox for running time?

### DISCUSSION THREAD

**Discussion by ea4k** [ea4k/klog#287] [Ideas]
Hello,
I would like to know what the users prefer, an icon or a checkbox. I need to take a decision and it is not completely clear for me:

This is how it looks with the icon:
![imagen](https://user-images.githubusercontent.com/29511627/121819214-2305c300-cc8c-11eb-963a-2e6eb1a8ae04.png)

This is how it looks with the checkbox:
![imagen](https://user-images.githubusercontent.com/29511627/121811448-bd521080-cc64-11eb-87e3-ee24ae0816f8.png)

What do you prefer?
Thanks for sharing your opinion!

**Comment by foldynl** [comment]
The pictures are the same

**Reply by ea4k** [reply]
You are right! O:-)

Just updated, thank you for the warning!

**Reply by foldynl** [reply]
And who can vote? ;-) Naturally the author must not vote ;-)

I would not be afraid of using icons when tooltips will be used - it will save a space, reduce the work of the translators and maybe it will be more intuitive.   

Otherwise, I sent a patch #288 to solve swapped icons symbols - my first implementation behaved strangely.
